assembl/static2/js/app/reducers/rootReducer.js

```diff
@@ -19,5 +20,6 @@ export default combineReducers({
   synthesis: Synthesis,
   auth: Auth,
   phase: Phase,
-  admin: Admin
+  admin: Admin,
+  ...screenDimensions
```

Comment:
why don't you use `combineReducers` in `screenDimensionsReducer.js` ?

Reply:
because I want screenWidth and screenHeight at the root of the store and not screenDimensions.width screenDimensions.height

Reply:
ok, I see